packages/shorebird_cli/test/src/commands/build/build_apk_command_test.dart

```diff
@@ -127,7 +127,7 @@ void main() {
         () => logger.info(
           '''
 📦 Generated an apk at:
-${lightCyan.wrap("./build/app/outputs/apk/release/app-release.apk")}''',
```

Comment:
I think these will fail on Windows.

Reply:
yeah good call will update the tests (although I'm sure that's not all that will fail on windows 🤣)

Reply:
done 👍